tests/bcachefs/checksum.ktest

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+#tester script to verify checksum correctness of bcachefs filesystem.
+#this script requires a bcachefs-tools installation, and a sufficient (1GB) amount of RAM to create a loopback filesystem
+
+#create neccessary infrastructure in tmpfs
+mkdir /tmp/bcachefs_checksum_test
+mount -t tmpfs none /tmp/bcachefs_checksum_test
+mkdir /tmp/bcachefs_checksum_test/mounted
+dd if=/dev/zero of=/tmp/bcachefs_checksum_test/zerofile.bin bs=4096 count=65536
+dd if=/dev/urandom of=/tmp/bcachefs_checksum_test/randfile.bin bs=4096 count=65536
+fallocate -l 1GiB /tmp/bcachefs_checksum_test/loopdir.bin
+uuid="$(bcachefs format /tmp/bcachefs_checksum_test/loopdir.bin | grep External\ UUID | rev | cut -d$'\t' -f1 | rev)"
```

Comment:
Cool, but this throws away all the other output from bcachefs format. Look at run_quiet as used by the other tests - this way we get the format output when running the tests in verbose mode.